README.md

```diff
@@ -1,44 +1,225 @@
-# User Manual
-  A detailed user manual for the buyer reference app is available [here](https://docs.google.com/document/d/1pGPZ0jwQH9AP0rdZXUcdv8B1QZudr86W3qjABsrlEso/edit?usp=sharing) 
-  
-# Introduction 
+# ONDC SDK BUYER APP
 
-This repo is ONDC Buyer App with microservice architecture
+A buyer app refers to any application that will help sellers interact with the buyers/ end consumers i.e., the demand side of any transaction, where the intent and transaction originate. These can include different types of applications including user experience (UX) based applications, voice assistants, chat-bots, etc. depicting the demand layer for the goods or services.
+
+## Documentation
+
+[Documentation](https://linktodocumentation)
+
+## Tech Stack
+
+**Client:** React JS, Redux, Javascript
+
+**Server:** Node JS, Express, Python, Nginx
+
+**Database:** MongoDB
+
+ONDC's buyer app uses a modern technical stack for web, mobile, and server-side development. The client layer of the web application is built using React JS v17, while the mobile app is built using React Native 18. Node JS v16 is used for server-side development, and Python v3.7 is used for the protocol layer.
```

Comment:
can you please mention in a table of
component|technology| version

Reply:
Done, Please check